src/main/scala-2/com/sksamuel/scapegoat/inspections/traits/AbstractTrait.scala

```diff
@@ -0,0 +1,35 @@
+package com.sksamuel.scapegoat.inspections.traits
+
+import com.sksamuel.scapegoat.*
```

Comment:
Otherwise the `import` fails on Scala 2.12.
```suggestion
import com.sksamuel.scapegoat._
```

Reply:
> Otherwise the `import` fails on Scala 2.12.

done. and ReadmeTest fixed too.